web/pageComponents/topicPages/FullWidthImage.tsx

```diff
@@ -9,17 +10,20 @@ type TeaserProps = {
 
 const FullWidthImage = ({ data, anchor }: TeaserProps) => {
   const { image, attribution, caption } = data.image
+  const { aspectRatio } = data.designOptions
+  const { width } = useWindowSize()
+
+  const ratio =
+    width && width < 750
+      ? Ratios.ONE_TO_TWO
+      : aspectRatio === String(Ratios.ONE_TO_TWO)
```

Comment:
this will always be false because Ratios.ONE_TO_TWO is '0.5' and your variable  is '2:1'